src/pgw_s5s8_proxy.erl

```diff
@@ -451,6 +466,15 @@ handle_response(ReqInfo, #gtp{version = v1} = Msg, Request, State, Data) ->
 handle_response(_, _Response, _Request, shutdown, _Data) ->
     keep_state_and_data;
 
+handle_response(#proxy_request{direction = sgw2pgw},
```

Comment:
Looks like this clause and clause at line 473 can be collected as:
```erlang
handle_response(#proxy_request{direction = sgw2pgw}, timeout, #gtp{type = create_session_request}, Req, _) 
  when Req =:= connecting; Req =:= connected ->
    keep_state_and_data;
```

Reply:
That is left over from handling them differently in an earlier version. I couldn't make my mind up if special handling will be needed or not (or might be needed in the future.
But it doesn't hurt to use the short version for now.